include/swift/AST/DiagnosticsDriver.def

```diff
@@ -42,91 +42,92 @@
 DIAG(REMARK,ID,Options,Text,Signature)
 #endif
 
-WARNING(warning_parallel_execution_not_supported,none,
+WARNING(warning_parallel_execution_not_supported, none,
```

Comment:
Please don't include formatting changes for code you're not touching.

Reply:
Oh, ok. I thought it'd be useful to do this as I was adding a new warning, but yeah, maybe not.